src/styles/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product {
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid white;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 19px 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    margin: 36px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $secondary;
+    margin: 4px 16px 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 16px 0;
+  }
+
+  &__review {
+    margin: 0;
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+  }
+
+  &__price-text {
+    margin: 0;
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__cost {
+    margin: 0;
+    color: $main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    width: calc(100% - 32px);
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead